src/iccpd/include/openbsd_tree.h

```diff
@@ -0,0 +1,571 @@
+/*	$OpenBSD: tree.h,v 1.14 2015/05/25 03:07:49 deraadt Exp $	*/
```

Comment:
Is there any copy right concerns of taking this code from git OpenBSD/src/sys/sys ?

Reply:
@gechiang: This particular code is also Opensource and is used in multiple places. Must not be a copyright concern.